Task 8 1 tournament creation page

f-lair opened the pull request:
The tournament creation page is added to the set of pages. In the first section, general information on the tournament (name, tournament mode, match mode) are entered. The second section provides a searchable multi-selection box, facilitating quick and easy selection of participants. It uses the '_tokenize2_' module. In the last section, tournament creation is confirmed by button press.

f-lair committed:
Initial commit

Added 'tournament_creation' page based on 'group_creation' page.

f-lair committed:
Added tournament_information section

Added 'tournament_information' section (1/3) to 'tournament_creation' page.

f-lair committed:
Included tokenize2 module

Included 'tokenize2' module, providing a fancy multi-selection box.

f-lair committed:
Added tournament_participants section

Added 'tournament_participants' section (2/3) to 'tournament_creation' page.

f-lair committed:
Added tournament_creation section

Added 'tournament_creation' section (3/3) to 'tournament_creation' page.

DavidRisch committed:
Removed css/tokenize2, replaced with use of npm dependency

DavidRisch commented on app.js:
```diff
@@ -23,6 +23,8 @@ app.use('/chosen', express.static('node_modules/chosen-js'))
 app.use('/jquery-ui', express.static('node_modules/jquery-ui-dist'))
 app.use('/hamburgers', express.static('node_modules/hamburgers/dist'))
+app.use('/bootstrap/css', express.static('node_modules/bootstrap/dist/css'))
+app.use('/tokenize2', express.static('node_modules/tokenize2/dist'))
```
Copying and modifying a css dependency works, but it make it hard to tell what was modified, updating the dependency becomes more difficult and it causes copyright/licensing problems.
It is best to use the dependency unmodified (I already implemented that in eb9dba7) and then have a separate css file (styles_general.css ?) with your modifications.
https://developer.mozilla.org/de/docs/Web/CSS/Spezifit%C3%A4t

f-lair replied:
I shortened the stylesheet (removed duplicate lines) and inserted it into the styles_general.css.

DavidRisch reviewed (changes requested):
Small addition: There should only be one `Initial commit` in the entire repo, the first commit of a branch should have a normal title describing the content. (No need to change it for this pr)

f-lair committed:
Inserted tokenize2 stylesheet into 'styles_general.css'

According to the former commit (eb9dba7923fe73ffd73e4e9638eed7403895e454), the stylesheet for tokenize2 multi-selection box is shortened and inserted into 'styles_general.css'.
